docs/quickstart.rst

```diff
@@ -61,11 +61,10 @@ ignored for now.)
 Preview your site
 -----------------
 
-Open a new terminal session and run the following commands to switch to your
-``output`` directory and launch Pelican's web server::
+Open a new terminal session and run the following commands to launch Pelican's
+web server::
 
-    cd ~/projects/yoursite/output
-    python -m pelican.server
+    pelican -l content
```

Comment:
`content` is redundant here.

Also there are a couple of assumptions (and possibly issues) here that are not mentioned:
 - if you define `OUTPUT_PATH` (to something non-default) in your settings and your settings is not called `pelicanconf.py` (default), this won't pickup your output path: you should supply the `-s configfile.py` as well
 - if you don't define `OUTPUT_PATH` but instead use `-o/--output`, similar situation: you should define same here as well
 - in most default(ish) cases, this should be run from the project root